docs/guidance/build/incremental_updates.md

```diff
@@ -1,8 +1,8 @@
-```{workedexample} Change history
+```{workedexample} Incremental updates
```

Comment:
This page isn't really about incremental updates. It's about the whole releases and records model. In fact, the examples actually show full updates.

Reply:
>  In fact, the examples actually show full updates.

Hmm, well, the contract and implementation releases don't include the info about the past stages. I guess we can change the award release to remove the tender section from there and we should be fine?

Reply:
Resolving as the page has been renamed.